"""
The easiest way to get an exception is to do something stupid
"""

print "hello world"
x = 5 / 0
lst = [1,2,3]
print(lst[3])
lst + 2
lst.add
d = {'a': 'hello'}
d['b']
print(this_is_not_a_var)

"""
hello world
Traceback (most recent call last):
  File "ex.py", line 2, in <module>
    x = 5 / 0
ZeroDivisionError: integer division or modulo by zero
"""
